addons/glusterfs/docker/entrypoint.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+import subprocess
+
+obj = json.load(open("/etc/fluid/config.json"))
+
+mount_point = obj["mounts"][0]["mountPoint"]
+target_path = obj["targetPath"]
+
+os.makedirs(target_path, exist_ok=True)
+
+if len(mount_point.split(":")) != 2:
+    print(
+        f"The mountPoint format [{mount_point}] is wrong, should be server:volumeId")
+    exit(1)
+
+server, volume_id = mount_point.split(":")
+args = ["glusterfs", "--volfile-server", server, "--volfile-id",
+        volume_id, target_path, "--no-daemon", "--log-file", "/dev/stdout"]
```

Comment:
May I know the reason of specifying the log file as /dev/stdout?

Reply:
By default, there is no output when mounting fails. So redirecting logs to stdout will be helpful when using "kubectl logs" for debug.